modules/mobile/docs/openapi.yaml

```diff
@@ -4022,10 +4022,10 @@ paths:
               examples:
                 Multiple matches:
                   value:
```

Comment:
These were just wrong. You can check the deployed version and see that it shows appointment examples instead of addresses.